econet_hvac_zone_template.yaml

```diff
@@ -0,0 +1,60 @@
+---
+# packages:
+#   econet: !include econet_hvac_furnace.yaml
+
+text_sensor:
+  - platform: econet
+    name: z${zoneid}name
+    sensor_datapoint: Z${zoneid}__NAME
+    request_mod: ${requestmod}
+    request_once: true
+
+climate:
+  - platform: econet
+    name: Zone ${zoneid}
+    visual:
+      min_temperature: "10"
+      max_temperature: "32.2222"
+      temperature_step: 1.0f
+    current_temperature_datapoint: ZON${zoneid}_SPT
+    target_temperature_low_datapoint: ZON${zoneid}HEAT
+    target_temperature_high_datapoint: ZON${zoneid}COOL
+    mode_datapoint: STATMODE
+    modes:
+      0: "HEAT"
+      1: "COOL"
+      2: "HEAT_COOL"
+      3: "FAN_ONLY"
+      4: "OFF"
+
+
+sensor:
+  - platform: econet
+    name: Zone ${zoneid} CFM
+    # id: z${zoneid}_cfm
+    sensor_datapoint: Z${zoneid}CFMDSP
+    request_mod: ${requestmod}
+    accuracy_decimals: 1
+    state_class: "measurement"
+    entity_category: "diagnostic"
+    unit_of_measurement: "cfm"
+  - platform: econet
+    name: Zone ${zoneid} humidity
+    sensor_datapoint: ZON${zoneid}HUMD
+    request_mod: ${requestmod}
+    accuracy_decimals: 1
+    state_class: "measurement"
+    entity_category: "diagnostic"
+
+
+
+binary_sensor:
+  - platform: econet
+    name: ZON${zoneid}ENAB
```

Comment:
This name will show up in the front end so it should be somehting more readable.

Reply:
fixed

Reply:
removed this sensor altogether.